test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -16,7 +17,7 @@
 TENSORFLOW2_VERSION = "2."
 
 
-TF_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="g3.8xlarge", processor="gpu")
+TF_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3.16xlarge", processor="gpu")
```

Comment:
This is a temporary change, because the TensorRT test failure only surfaced on non-g3 instances.